frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -647,29 +648,46 @@ export function getMappingInfoForSupportedLayer(state: OxalisState): ActiveMappi
   );
 }
 
-function _getTransformsForLayerOrNull(layer: APIDataLayer): Matrix4x4 | null {
-  if (!layer.coordinateTransformations) {
+type Transform =
+  | { type: "affine"; affineMatrix: Matrix4x4 }
+  | { type: "thin_plate_spline"; affineMatrix: Matrix4x4; tpsInv: TPS3D };
```

Comment:
Just noting the use of snake-case instead of camel-case. Ignore, if it was deliberate.

Reply:
It felt natural to me, since these also are in the datasource-properties.json. I'm open to changes, but it was deliberate, at least :)